README.md

```diff
@@ -104,6 +104,7 @@ https://github.com/readium/r2-opds-js/blob/develop/src/opds/init-globals.ts
 ```typescript
 // npm install r2-opds-js
 // "@opds" is a dist path alias, for example EcmaScript6/ES2015 'node_modules/r2-opds-js/dist/es6-es2015/src/opds/'
+// Tips!: see https://www.npmjs.com/package/tsconfig-paths to execute the compiled files with node
```

Comment:
When you say "the compiled files", do you actually mean the "uncompiled" / "non-transpiled" TS files?

Note that `r2-opds-js` has a basic `ts-node` setup for unit tests (via Ava). As you can see, we already use `tsconfig-paths`:

https://github.com/readium/r2-opds-js/blob/e410866da1483632c756d9f3a3f1f09bb1666bbd/package.json#L72-L73

https://github.com/readium/r2-opds-js/blob/e410866da1483632c756d9f3a3f1f09bb1666bbd/package.json#L119

https://github.com/readium/r2-opds-js/blob/e410866da1483632c756d9f3a3f1f09bb1666bbd/ava.config.js#L38-L49

Same with `r2-shared-js`:

https://github.com/readium/r2-shared-js/blob/18da7da29a3649e62915abd32329b9b5eb5e731b/package.json#L81-L82

https://github.com/readium/r2-shared-js/blob/18da7da29a3649e62915abd32329b9b5eb5e731b/package.json#L132

https://github.com/readium/r2-shared-js/blob/18da7da29a3649e62915abd32329b9b5eb5e731b/ava.config.js#L38-L49

Reply:
can you add it in the readme ? , whatever the way

Reply:
Should all `r2-xxx-js` repositories be updated? (i.e. `r2-utils-js`, `r2-lcp-js`, `r2-shared-js`, `r2-opds-js`, `r2-streamer-js`, `r2-navigator-js`, `r2-testapp-js`)